tailwind.config.ts

```diff
@@ -17,6 +17,7 @@ const config: Partial<Config> = {
         tracklist: "min-content auto auto auto min-content",
         tilesWide: "repeat(auto-fit, minmax(350px, 1fr))",
         tilesNarrow: "repeat(auto-fit, minmax(200px, 1fr))",
+        tilesOneLine: "repeat(6, minmax(220px, 1fr))",
```

Comment:
Where do you get the number 6 from?

Reply:
I don't see us ever having more than 6 tiles. It could just as well be 10.